packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1270,6 +1270,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             getNodePackagePath: async (nodePath: string) => this.getGCNodePackagePath(nodePath),
             getLastSummaryTimestampMs: () => this.messageAtLastSummary?.timestamp,
             readAndParseBlob: async <T>(id: string) => readAndParse<T>(this.storage, id),
+            snapshotCacheExpiryMs: _storage.policies?.maximumCacheDurationMs,
```

Comment:
This is the line causing the test failure.  `_storage` is a `ContainerStorageAdapter` which runs the code that throws in order to look up the policies (it's a Lazy-type implementation).

Reply:
Yeah... we're not attached when we get here, and so there is no storage service yet :(  I'll have to think about how else to get this value plumbed through.  @vladsud any immediate thoughts?  I'm in a meeting so going to come back to this in a bit

Reply:
Maybe I can change that param to be a getter that's invoked later...?

Reply:
Ok talked with Navin - Create Detached is irrelevant - this value is only needed for Summarizer which always loads an existing container from the server and driver policy will be available.

As for the test failure - the fix will be to downgrade the NoRealStorageInDetachedContainer log to generic - since we also throw, callers will decide whether to log a proper error (we won't in this case).  And GC will have to handle the fact that this can be undefined, knowing it won't interfere with actual Summarizer trying to run Sweep.